### PR TITLE
Add basic UI

### PR DESCRIPTION
**Pull request by Talha-Hanif5666**
### In this PR, I have completed the following tasks:

- [ ] Create an app.rb file that will serve as your console app entry point. It should have methods that do the following:
- [ ] List all books.✔
- [ ] List all people.✔
- [ ] Create a person (teacher or student, not a plain Person).✔
- [ ] Create a book.✔
- [ ] Create a rental.✔
- [ ] List all rentals for a given person id.✔

### CODE REVIEW

File: rb-files/app.rb [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+      permission = permission == 'Y'
+      #   @people << Student.new(nil, age, name, permission)
+      print 'What is the student\'s classroom? '
+      classroom = gets.chomp
+      @people.push(Student.new(age, name, permission, classroom))
+    when 2
+      print 'Specialization: '
+      specialization = gets.chomp
+      @people << Teacher.new(age, name, specialization)
+    end
```
Comment (AmaduKamara) [inline rb-files/app.rb:58]: - [ ] Creating a Teacher feature is not working as expected. When I enter the specialization for the Teacher, the app crashes with an error shown in the screenshot below. Please fix that so a teacher is created successfully just as the student creation. 👍🏽
- [ ] 
Screenshot 2023-04-13 at 6 56 49 PM

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️

Hi @Talha-Hanif5666,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽
✅ Good descriptive PR 👍🏽
✅ Nice UI display 👍🏽

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly add to your README the command to start the app,`ruby ./rb-files/main.rb`, so your README looks more documented 👍🏽
- [ ] Kindly work on the feature for creating a Teacher and add the command to start the app on your README 👍🏽
- [ ] Everything else works fine 👍🏽

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: rb-files/app.rb (hunk 1 above)
Comment (Whoistolu) [reply rb-files/app.rb:58]: - Well resolved @Talha-Hanif5666

**Review by Whoistolu — APPROVED** [review]
### Hi @Talha-Hanif5666 

Great job fixing the recommended changes by the previous reviewer 👍🏽 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**